packages/arb-token-bridge-ui/src/components/App/AppConnectionFallbackContainer.tsx

```diff
@@ -1,20 +1,19 @@
 import { ExternalLink } from '../common/ExternalLink'
+import Image, { ImageProps } from 'next/image'
```

Comment:
please rearrange the imports

Reply:
We should set prettier/eslint for that. I will use VSCode "sort import" for now.